Code-the-Dream-School/hh-team4-front
# 26 edit page

## Description

_1) What does this PR do? 2) List out what you did._

It edits the inventory, and added a few changes to the add drugs page.

## Related Issue

_Provide a link to the related issue 

## Type of Changes

_Feature update/change/addition, bug fix, styling update, etc._

## Acceptance Criteria

_Include the checked-off acceptance criteria from the issue._

## Update Screenshots

_Provide screenshots of the view before and after the changes._
<img width="421" alt="Screenshot 2024-12-13 at 10 01 19 PM" src="https://github.com/user-attachments/assets/ea981735-e85a-49b7-9bff-00914244a37d" />

### Before

### After

## Testing Instructions

_In a list format, tell us how to test this PR._

-First make sure you can add the medicine.
-Second make sure it was added by checking the list of the inventory on the dashboard.
-Third click the edit button and you can update the changes. 

## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._

I learned how to use postman with fetch.
